docs/template.md

```diff
@@ -1,7 +1,25 @@
-# qgrid 
+# qgrid template
 ## markup
 ```html
-<q-grid rows='data'></q-grid>
+<q-grid rows="data" >
+        <q-grid:head> 
+            <q-grid:td> 
+                <i title="{{$column.name}}">{{$column.label}}</i>
+            </q-grid:td>
+        </q-grid:head>
+        <q-grid:body>
+            <q-grid:tr>
+            </q-grid:tr>
+            <q-grid:td key="firstName">
+                <b>{{$value}}</b>
+            </q-grid:td>
+            <q-grid:td key="dateBirth">
```

Comment:
Of course, if it's a watcher, it will affect performance badly. Probably, we can have only restricted set of events like 'onRowSelect', 'onModeChanged' yo reduce load.